package.json

```diff
@@ -1,29 +1,8 @@
 {
   "name": "agenda",
-  "version": "0.9.1",
   "description": "Light weight job scheduler for Node.js",
-  "main": "index.js",
-  "scripts": {
-    "test": "eslint lib/* index.js && mocha --reporter spec --timeout 3500",
-    "blanket": {
-      "pattern": "lib",
-      "data-cover-never": "node_modules"
-    }
-  },
-  "repository": {
-    "type": "git",
-    "url": "git://github.com/agenda/agenda"
-  },
-  "keywords": [
-    "job",
-    "jobs",
-    "cron",
-    "delayed",
-    "scheduler",
-    "runner"
-  ],
+  "version": "0.9.1",
```

Comment:
I'd suggest fixing up the `package.json` so the reordering of fields doesn't get merged as it's not related to this PR.